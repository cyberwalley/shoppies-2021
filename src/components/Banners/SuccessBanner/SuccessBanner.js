import {Banner, List} from '@shopify/polaris';
import './SuccessBanner.css'
export const SuccessBanner = () => {
  return (
    <Banner
      title="You have nominated 5 movies."
      status="success"
    >
      <List>
        <List.Item>
          You are only allowed a maximum of 5 nominations.
        </List.Item>
        <List.Item>
          You can go to the nomination tab to review or replace your nominees
        </List.Item>
      </List>
    </Banner>
  )
}
