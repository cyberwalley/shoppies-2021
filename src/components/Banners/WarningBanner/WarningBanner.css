.Polaris-Banner{
  margin-bottom: 20px;
}