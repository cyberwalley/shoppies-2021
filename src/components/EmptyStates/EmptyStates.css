.Polaris-EmptyState__ImageContainer img {
  width: 450px !important;
}