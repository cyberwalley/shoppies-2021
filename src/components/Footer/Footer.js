import React from 'react'
import {Link, Page, Stack} from "@shopify/polaris";
export const Footer = ({children}) => {
  return (
    <footer className="footer-wrapper_outer">
      <div className="footer-wrapper_inner">
        <Page fullWidth>
          <Stack distribution="center">
          <p>Learn more about {""}
            <Link url="https://github.com/cyberwalley/shoppies-2021" external>Shoppies</Link>
          </p>
          </Stack>
        </Page>
      </div>
    </footer>
  )
}
