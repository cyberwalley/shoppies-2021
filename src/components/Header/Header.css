.Polaris-TopBar{
  background-color: var(--color-green);
}