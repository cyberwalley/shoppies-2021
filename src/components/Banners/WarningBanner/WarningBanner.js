import {Banner, List} from '@shopify/polaris';
import './WarningBanner.css'
export const WarningBanner = ({searchTerm}) => {
  return (
    <Banner
      title={searchTerm}
      status="warning"
    >
      <List>
        <List.Item>
          Check your spelling and try again.
        </List.Item>
      </List>
    </Banner>
  )
}
