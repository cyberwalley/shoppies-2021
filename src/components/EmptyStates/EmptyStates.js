import {EmptyState} from '@shopify/polaris';
import './EmptyStates.css'
export const EmptyStates = ({heading, subheading, image }) => {
  return (
    <EmptyState
      heading={heading}
      image={image}
    >
      <p>{subheading}</p>
    </EmptyState>
  )
}
