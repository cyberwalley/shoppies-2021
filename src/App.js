import './Variables.css';
import {Header} from './components/Header'
import {Main} from './components/Main'
import {Footer} from './components/Footer'
import {Search} from './components/Search'
import {Tab} from './components/Tab'
import { useState} from "react";
import debounce from 'lodash.debounce';
import {Page, Card} from '@shopify/polaris';
import React from 'react'

function App() {
  const [inputValue, setInputValue] = useState('');
  const [searchValue, setSearchValue] = useState("");

  // search and debounce
  const updateValue = (newValue) =>{
    setInputValue(newValue);
    debouncedSearch(newValue);
  }

  const debouncedSearch = React.useMemo(
    () => debounce(newValue => { setSearchValue(newValue)}, 850), []
  )

  return (
    <div className="site-wrapper">
      <Header />
      <Main>
        <Search value={inputValue} onChange={(event) => updateValue(event)}/>
        <Page>
          <Card sectioned>
            <Tab searchValue={searchValue}/>
          </Card>
        </Page>
      </Main>
      <Footer />
    </div>
  );
}

export default App;
