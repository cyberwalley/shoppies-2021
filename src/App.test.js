Object.defineProperty(window, 'matchMedia', {
  writable: true,
  value: jest.fn().mockImplementation(query => ({
    matches: false,
    media: query,
    onchange: null,
    onAction: null,
    addListener: jest.fn(), // Deprecated
    removeListener: jest.fn(), // Deprecated
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    dispatchEvent: jest.fn(),
  })),
});

import React from "react";
import './setupTest.js'
import { shallow, mount } from 'enzyme';

import toJson from 'enzyme-to-json';
import App from './App';
import {Header} from './components/Header'
import {Main} from './components/Main'
import {Footer} from './components/Footer'
import {Search} from './components/Search'
import {Tab} from './components/Tab'
import {MovieCard} from './components/MovieCard'
import {EmptyStates} from './components/EmptyStates'
import {MoreMovieDetails} from './components/MoreMovieDetails'
import {Skeleton} from './components/Skeleton'
import {Spinners} from './components/Spinners'
import {Modals} from './components/Modals'
import {SuccessBanner} from './components/Banners/SuccessBanner'
import {WarningBanner} from './components/Banners/WarningBanner'


const movieTitle= {
  movieOne: 'Dracula',
  movieTwo: 'Harry potter',
  movieThree: 'Spiderman'
}

describe('rendering components', ()=> {
  it("renders App without crashing", () => {
    shallow(<App />);
  });
  it("renders SuccessBanner without crashing", () => {
    shallow(<SuccessBanner />);
  });
  it("renders WarningBanner without crashing", () => {
    shallow(<WarningBanner />);
  });
  it("renders EmptyStates without crashing", () => {
    shallow(<EmptyStates />);
  });
  it("renders Footer without crashing", () => {
    shallow(<Footer />);
  });
  it("renders Header without crashing", () => {
    shallow(<Header />);
  });
  it("renders Main without crashing", () => {
    shallow(<Main />);
  });
  it("renders Modals without crashing", () => {
    shallow(<Modals />);
  });
  it("renders Search without crashing", () => {
    shallow(<Search />);
  });
  it("renders Skeleton without crashing", () => {
    shallow(<Skeleton />);
  });
  it("renders Spinners without crashing", () => {
    shallow(<Spinners />);
  });
  it("renders Tab without crashing", () => {
    shallow(<Tab />);
  });
}); 


describe('snapshots', ()=> {
  it("App snapshots", () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("SuccessBanner snapshots", () => {
    const tree = shallow(<SuccessBanner />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("WarningBanner snapshots", () => {
    const tree = shallow(<WarningBanner />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("EmptyStates snapshots", () => {
    const tree = shallow(<EmptyStates />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Footer snapshots", () => {
    const tree = shallow(<Footer />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Header snapshots", () => {
    const tree = shallow(<Header />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Main snapshots", () => {
    const tree = shallow(<Main />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Modals snapshots", () => {
    const tree = shallow(<Modals />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Search snapshots", () => {
    const tree = shallow(<Search />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Skeleton snapshots", () => {
    const tree = shallow(<Skeleton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Spinners snapshots", () => {
    const tree = shallow(<Spinners />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  it("Tab snapshots", () => {
    const tree = shallow(<Tab />);
    expect(toJson(tree)).toMatchSnapshot();
  });
  
}); 

describe('passing props', ()=> {
  const tabWrapper = mount(<Tab searchValue={movieTitle} />);
   it("accepts movie data", () => {
    expect(tabWrapper.props().searchValue).toEqual(movieTitle);
  }); 
}); 