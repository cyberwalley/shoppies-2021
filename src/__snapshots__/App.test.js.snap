// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots App snapshots 1`] = `
<div
  className="site-wrapper"
>
  <Header />
  <Main>
    <Search
      onChange={[Function]}
      value=""
    />
    <Page>
      <Card
        sectioned={true}
      >
        <Tab
          searchValue=""
        />
      </Card>
    </Page>
  </Main>
  <Footer />
</div>
`;

exports[`snapshots EmptyStates snapshots 1`] = `
<EmptyState>
  <p />
</EmptyState>
`;

exports[`snapshots Footer snapshots 1`] = `
<footer
  className="footer-wrapper_outer"
>
  <div
    className="footer-wrapper_inner"
  >
    <Page
      fullWidth={true}
    >
      <Memo(Stack)
        distribution="center"
      >
        <p>
          Learn more about 
          <Link
            external={true}
            url="https://github.com/cyberwalley/shoppies-2021"
          >
            Shoppies
          </Link>
        </p>
      </Memo(Stack)>
    </Page>
  </div>
</footer>
`;

exports[`snapshots Header snapshots 1`] = `
<header>
  <TopBar
    theme="https://cdn.shopify.com/s/files/1/2506/6936/files/shoppies-logo.svg?v=1609786867"
  />
</header>
`;

exports[`snapshots Main snapshots 1`] = `<main />`;

exports[`snapshots Modals snapshots 1`] = `
<Modal
  primaryAction={
    Object {
      "content": "Close",
      "onAction": [Function],
    }
  }
  title=""
>
  <Section$3 />
</Modal>
`;

exports[`snapshots Search snapshots 1`] = `
<div
  className="hero-wrapper"
>
  <Layout
    sectioned={true}
  >
    <Page>
      <Heading
        element="h1"
      >
        Nominate your favorite movies
      </Heading>
      <p
        className="mega-subtitle"
      >
        Popular keywords: Harry potter, Dracula, Spiderman...
      </p>
      <TextField
        autoComplete={false}
        inputMode="text"
        placeholder="Type movie title here..."
        prefix={
          <Icon
            color="base"
            source={[Function]}
          />
        }
        style={
          Object {
            "height": "225px",
          }
        }
        type="search"
      />
    </Page>
  </Layout>
</div>
`;

exports[`snapshots Skeleton snapshots 1`] = `
<SkeletonPage>
  <Layout>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
    <Section$4
      oneThird={true}
    >
      <Card
        sectioned={true}
      >
        <TextContainer>
          <SkeletonDisplayText
            size="extraLarge"
          />
          <SkeletonBodyText />
        </TextContainer>
      </Card>
    </Section$4>
  </Layout>
</SkeletonPage>
`;

exports[`snapshots Spinners snapshots 1`] = `
<Spinner
  accessibilityLabel="Spinner example"
  size="large"
/>
`;

exports[`snapshots SuccessBanner snapshots 1`] = `
<ForwardRef(Banner)
  status="success"
  title="You have nominated 5 movies."
>
  <List>
    <Item$6>
      You are only allowed a maximum of 5 nominations.
    </Item$6>
    <Item$6>
      You can go to the nomination tab to review or replace your nominees
    </Item$6>
  </List>
</ForwardRef(Banner)>
`;

exports[`snapshots Tab snapshots 1`] = `
<AppProvider$1
  i18n={
    Object {
      "Polaris": Object {
        "ActionMenu": Object {
          "RollupActions": Object {
            "rollupButton": "Actions",
          },
        },
        "Actions": Object {
          "moreActions": "More actions",
        },
        "Autocomplete": Object {
          "spinnerAccessibilityLabel": "Loading",
        },
        "Avatar": Object {
          "label": "Avatar",
          "labelWithInitials": "Avatar with initials {initials}",
        },
        "Badge": Object {
          "PROGRESS_LABELS": Object {
            "complete": "Complete",
            "incomplete": "Incomplete",
            "partiallyComplete": "Partially complete",
          },
          "STATUS_LABELS": Object {
            "attention": "Attention",
            "critical": "Critical",
            "info": "Info",
            "new": "New",
            "success": "Success",
            "warning": "Warning",
          },
          "progressAndStatus": "{statusLabel} {progressLabel}",
        },
        "Button": Object {
          "connectedDisclosureAccessibilityLabel": "Related actions",
          "spinnerAccessibilityLabel": "Loading",
        },
        "Common": Object {
          "cancel": "Cancel",
          "checkbox": "checkbox",
          "clear": "Clear",
          "close": "Close",
          "more": "More",
          "newWindowAccessibilityHint": "(opens a new window)",
          "submit": "Submit",
          "undo": "Undo",
        },
        "ContextualSaveBar": Object {
          "discard": "Discard",
          "save": "Save",
        },
        "DataTable": Object {
          "navAccessibilityLabel": "Scroll table {direction} one column",
          "sortAccessibilityLabel": "sort {direction} by",
          "totalRowHeading": "Total",
          "totalsRowHeading": "Totals",
        },
        "DatePicker": Object {
          "days": Object {
            "friday": "Friday",
            "monday": "Monday",
            "saturday": "Saturday",
            "sunday": "Sunday",
            "thursday": "Thursday",
            "tuesday": "Tuesday",
            "wednesday": "Wednesday",
          },
          "daysAbbreviated": Object {
            "friday": "Fr",
            "monday": "Mo",
            "saturday": "Sa",
            "sunday": "Su",
            "thursday": "Th",
            "tuesday": "Tu",
            "wednesday": "We",
          },
          "end": "End of range",
          "months": Object {
            "april": "April",
            "august": "August",
            "december": "December",
            "february": "February",
            "january": "January",
            "july": "July",
            "june": "June",
            "march": "March",
            "may": "May",
            "november": "November",
            "october": "October",
            "september": "September",
          },
          "nextMonth": "Show next month, {nextMonth} {nextYear}",
          "previousMonth": "Show previous month, {previousMonthName} {showPreviousYear}",
          "start": "Start of range",
          "today": "Today ",
        },
        "DiscardConfirmationModal": Object {
          "message": "If you discard changes, youâ€™ll delete any edits you made since you last saved.",
          "primaryAction": "Discard changes",
          "secondaryAction": "Continue editing",
          "title": "Discard all unsaved changes",
        },
        "DropZone": Object {
          "allowMultiple": Object {
            "actionHintFile": "or drop files to upload",
            "actionHintImage": "or drop images to upload",
            "actionTitleFile": "Add files",
            "actionTitleImage": "Add images",
            "labelFile": "Upload files",
            "labelImage": "Upload images",
            "overlayTextFile": "Drop files to upload",
            "overlayTextImage": "Drop images to upload",
          },
          "errorOverlayTextFile": "File type is not valid",
          "errorOverlayTextImage": "Image type is not valid",
          "single": Object {
            "actionHintFile": "or drop file to upload",
            "actionHintImage": "or drop image to upload",
            "actionTitleFile": "Add file",
            "actionTitleImage": "Add image",
            "labelFile": "Upload file",
            "labelImage": "Upload image",
            "overlayTextFile": "Drop file to upload",
            "overlayTextImage": "Drop image to upload",
          },
        },
        "EmptySearchResult": Object {
          "altText": "Empty search results",
        },
        "Filters": Object {
          "cancel": "Cancel",
          "clear": "Clear",
          "clearAllFilters": "Clear all filters",
          "clearLabel": "Clear {filterName}",
          "done": "Done",
          "filter": "Filter {resourceName}",
          "moreFilters": "More filters",
          "moreFiltersWithCount": "More filters ({count})",
          "noFiltersApplied": "No filters applied",
        },
        "Frame": Object {
          "Navigation": Object {
            "closeMobileNavigationLabel": "Close navigation",
          },
          "navigationLabel": "Navigation",
          "skipToContent": "Skip to content",
        },
        "IndexProvider": Object {
          "a11yCheckboxDeselectAllMultiple": "Deselect all {itemsLength} {resourceNamePlural}",
          "a11yCheckboxDeselectAllSingle": "Deselect {resourceNameSingular}",
          "a11yCheckboxSelectAllMultiple": "Select all {itemsLength} {resourceNamePlural}",
          "a11yCheckboxSelectAllSingle": "Select {resourceNameSingular}",
          "allItemsSelected": "All {itemsLength}+ {resourceNamePlural} are selected.",
          "defaultItemPlural": "Items",
          "defaultItemSingular": "Item",
          "selected": "{selectedItemsCount} selected",
        },
        "IndexTable": Object {
          "emptySearchDescription": "Try changing the filters or search term",
          "emptySearchTitle": "No {resourceNamePlural} found",
          "onboardingBadgeText": "New",
          "resourceLoadingAccessibilityLabel": "Loading {resourceNamePlural}â€¦",
          "selectAllItems": "Select all {itemsLength}+ {resourceNamePlural}",
          "selectAllLabel": "Select all {resourceNamePlural}",
          "selectButtonText": "Select",
          "selectItem": "Select {resourceName}",
          "selected": "{selectedItemsCount} selected",
          "undo": "Undo",
        },
        "Loading": Object {
          "label": "Page loading bar",
        },
        "MediaCard": Object {
          "popoverButton": "Actions",
        },
        "Modal": Object {
          "iFrameTitle": "body markup",
          "modalWarning": "These required properties are missing from Modal: {missingProps}",
        },
        "Pagination": Object {
          "next": "Next",
          "pagination": "Pagination",
          "previous": "Previous",
        },
        "ProgressBar": Object {
          "exceedWarningMessage": "Values passed to the progress prop shouldnâ€™t exceed 100. Setting {progress} to 100.",
          "negativeWarningMessage": "Values passed to the progress prop shouldnâ€™t be negative. Resetting {progress} to 0.",
        },
        "ResourceList": Object {
          "BulkActions": Object {
            "actionsActivatorLabel": "Actions",
            "moreActionsActivatorLabel": "More actions",
            "warningMessage": "To provide a better user experience. There should only be a maximum of {maxPromotedActions} promoted actions.",
          },
          "DateSelector": Object {
            "FilterLabelForValue": Object {
              "coming_month": "next month",
              "coming_quarter": "in the next 3 months",
              "coming_week": "next week",
              "coming_year": "in the next year",
              "on_or_after": "after {date}",
              "on_or_before": "before {date}",
              "past_month": "in the last month",
              "past_quarter": "in the last 3 months",
              "past_week": "in the last week",
              "past_year": "in the last year",
            },
            "SelectOptions": Object {
              "ComingMonth": "next month",
              "ComingQuarter": "in the next 3 months",
              "ComingWeek": "next week",
              "ComingYear": "in the next year",
              "OnOrAfter": "on or after",
              "OnOrBefore": "on or before",
              "PastMonth": "in the last month",
              "PastQuarter": "in the last 3 months",
              "PastWeek": "in the last week",
              "PastYear": "in the last year",
            },
            "dateFilterLabel": "Select a value",
            "dateValueError": "Match YYYY-MM-DD format",
            "dateValueLabel": "Date",
            "dateValuePlaceholder": "YYYY-MM-DD",
          },
          "FilterControl": Object {
            "textFieldLabel": "Search {resourceNamePlural}",
          },
          "FilterCreator": Object {
            "addFilterButtonLabel": "Add filter",
            "filterButtonLabel": "Filter",
            "selectFilterKeyPlaceholder": "Select a filterâ€¦",
            "showAllWhere": "Show all {resourceNamePlural} where:",
          },
          "FilterValueSelector": Object {
            "selectFilterValuePlaceholder": "Select a filterâ€¦",
          },
          "Item": Object {
            "actionsDropdown": "Actions dropdown",
            "actionsDropdownLabel": "Actions for {accessibilityLabel}",
            "viewItem": "View details for {itemName}",
          },
          "a11yCheckboxDeselectAllMultiple": "Deselect all {itemsLength} {resourceNamePlural}",
          "a11yCheckboxDeselectAllSingle": "Deselect {resourceNameSingular}",
          "a11yCheckboxSelectAllMultiple": "Select all {itemsLength} {resourceNamePlural}",
          "a11yCheckboxSelectAllSingle": "Select {resourceNameSingular}",
          "allFilteredItemsSelected": "All {itemsLength}+ {resourceNamePlural} in this filter are selected.",
          "allItemsSelected": "All {itemsLength}+ {resourceNamePlural} in your store are selected.",
          "ariaLivePlural": "{itemsLength} items",
          "ariaLiveSingular": "{itemsLength} item",
          "defaultItemPlural": "items",
          "defaultItemSingular": "item",
          "emptySearchResultDescription": "Try changing the filters or search term",
          "emptySearchResultTitle": "No {resourceNamePlural} found",
          "loading": "Loading {resource}",
          "selectAllFilteredItems": "Select all {itemsLength}+ {resourceNamePlural} in this filter",
          "selectAllItems": "Select all {itemsLength}+ {resourceNamePlural} in your store",
          "selectButtonText": "Select",
          "selected": "{selectedItemsCount} selected",
          "showing": "Showing {itemsCount} {resource}",
          "showingTotalCount": "Showing {itemsCount} of {totalItemsCount} {resource}",
          "sortingLabel": "Sort by",
        },
        "SkeletonPage": Object {
          "loadingLabel": "Page loading",
        },
        "Tabs": Object {
          "toggleTabsLabel": "More tabs",
        },
        "Tag": Object {
          "ariaLabel": "Remove {children}",
        },
        "TextField": Object {
          "characterCount": "{count} characters",
          "characterCountWithMaxLength": "{count} of {limit} characters used",
        },
        "TooltipOverlay": Object {
          "accessibilityLabel": "Tooltip: {label}",
        },
        "TopBar": Object {
          "SearchField": Object {
            "clearButtonLabel": "Clear",
            "search": "Search",
          },
          "toggleMenuLabel": "Toggle menu",
        },
        "VideoThumbnail": Object {
          "playButtonA11yLabel": Object {
            "default": "Play video",
            "defaultWithDuration": "Play video of length {duration}",
            "duration": Object {
              "hours": Object {
                "one": Object {
                  "andMinute": "{hourCount} hour and {minuteCount} minute",
                  "andMinutes": "{hourCount} hour and {minuteCount} minutes",
                  "andSecond": "{hourCount} hour and {secondCount} second",
                  "andSeconds": "{hourCount} hour and {secondCount} seconds",
                  "minuteAndSecond": "{hourCount} hour, {minuteCount} minute, and {secondCount} second",
                  "minuteAndSeconds": "{hourCount} hour, {minuteCount} minute, and {secondCount} seconds",
                  "minutesAndSecond": "{hourCount} hour, {minuteCount} minutes, and {secondCount} second",
                  "minutesAndSeconds": "{hourCount} hour, {minuteCount} minutes, and {secondCount} seconds",
                  "only": "{hourCount} hour",
                },
                "other": Object {
                  "andMinute": "{hourCount} hours and {minuteCount} minute",
                  "andMinutes": "{hourCount} hours and {minuteCount} minutes",
                  "andSecond": "{hourCount} hours and {secondCount} second",
                  "andSeconds": "{hourCount} hours and {secondCount} seconds",
                  "minuteAndSecond": "{hourCount} hours, {minuteCount} minute, and {secondCount} second",
                  "minuteAndSeconds": "{hourCount} hours, {minuteCount} minute, and {secondCount} seconds",
                  "minutesAndSecond": "{hourCount} hours, {minuteCount} minutes, and {secondCount} second",
                  "minutesAndSeconds": "{hourCount} hours, {minuteCount} minutes, and {secondCount} seconds",
                  "only": "{hourCount} hours",
                },
              },
              "minutes": Object {
                "one": Object {
                  "andSecond": "{minuteCount} minute and {secondCount} second",
                  "andSeconds": "{minuteCount} minute and {secondCount} seconds",
                  "only": "{minuteCount} minute",
                },
                "other": Object {
                  "andSecond": "{minuteCount} minutes and {secondCount} second",
                  "andSeconds": "{minuteCount} minutes and {secondCount} seconds",
                  "only": "{minuteCount} minutes",
                },
              },
              "seconds": Object {
                "one": "{secondCount} second",
                "other": "{secondCount} seconds",
              },
            },
          },
        },
      },
    }
  }
>
  <Tabs
    fitted={true}
    key="movies-fitted-2"
    onSelect={[Function]}
    selected={0}
    tabs={
      Array [
        Object {
          "accessibilityLabel": "Movies",
          "content": <span>
            Movies
             
            <Badge
              status="new"
            >
              0
            </Badge>
          </span>,
          "contentMarkUp": <EmptyStates
            heading="Search for your favorite movie by name."
            image="https://cdn.shopify.com/s/files/1/2506/6936/files/movies-empty-state.svg?v=1609784787"
            subheading="The search result will appear here."
          />,
          "id": "movies-fitted-2",
          "panelID": "movies-fitted-content-2",
          "title": "Movies",
        },
        Object {
          "content": <span>
            Nomination
             
            <Badge
              status="new"
            >
              0
            </Badge>
          </span>,
          "contentMarkUp": <EmptyStates
            heading="Nominate a movie and it will show up here."
            image="https://cdn.shopify.com/s/files/1/2506/6936/files/nomination-empty-state-2.svg?v=1609785285"
            subheading="Nominate a movie and it will show up here."
          />,
          "id": "nomination-fitted-2",
          "panelID": "nomination-fitted-Ccontent-2",
          "title": "Nomination",
        },
      ]
    }
  >
    <Section$2
      title="Movies"
    >
      <Layout>
        <EmptyStates
          heading="Search for your favorite movie by name."
          image="https://cdn.shopify.com/s/files/1/2506/6936/files/movies-empty-state.svg?v=1609784787"
          subheading="The search result will appear here."
        />
      </Layout>
    </Section$2>
  </Tabs>
  <Modals
    onClose={[Function]}
    open={false}
    spinning={false}
  />
</AppProvider$1>
`;

exports[`snapshots WarningBanner snapshots 1`] = `
<ForwardRef(Banner)
  status="warning"
>
  <List>
    <Item$6>
      Check your spelling and try again.
    </Item$6>
  </List>
</ForwardRef(Banner)>
`;
